targets/nvidia-jetson-orin.nix

```diff
@@ -51,13 +42,17 @@
           ../modules/virtualization/microvm/microvm-host.nix
           ../modules/virtualization/microvm/netvm.nix
           {
+            users.users."ghaf".extraGroups = ["docker"];
             ghaf = {
               hardware.nvidia.orin.enable = true;
               hardware.nvidia.orin.somType = som;
-
+              #virtualization.docker.daemon.enable = true;
               virtualization.microvm-host.enable = true;
               host.networking.enable = true;
 
+              virtualization.microvm.netvm.enable = true;
+              virtualization.microvm.netvm.extraModules = netvmExtraModules;
+
```

Comment:
Should we also remove this https://github.com/tiiuae/ghaf/pull/259/files#diff-5e647cec450851446913497aec8e4d19c2440a05aea244e642b0048269d51fbeL106 as redundant in respect to above line?

Reply:
`virtualization.microvm.netvm.extraModules = netvmExtraModules;` <- this needs to be set in both files. The NixOS module system will then combine the two lists and they are added as a two separate modules into the netvm configuration imports.

Reply:
The separation in general between the targets and the modules which I would like to see, is that the targets would contain more, well, example targets which contain more target-specific information, depending on usecase. Some might want to pass through wifi-card and others might not.

Modules should then aim to provide more general modules for use in other projects/flakes, through flake system. For example one might want to create project for Orin NX and not use any VMs at all, when soneone other might want to run it with all the bells and whistles included, and modules should provide easy way to enable/disable various features.

Currently this separation is not very nicely and clearly happening, but we can leave the refactoring and cleanup to further commits.

Reply:
It was left as it was required in two different files to make netvmExtraModules section merge.